src/System.Runtime.Extensions/src/System/AppDomain.cs

```diff
@@ -280,5 +280,144 @@ public void SetThreadPrincipal(IPrincipal principal)
                 _defaultPrincipal = principal;
             }
         }
+
+        public ObjectHandle CreateInstance(string assemblyName, string typeName)
+        {
+            // jit does not check for that, so we should do it ...
+            if (this == null)
```

Comment:
@jkotas is this useful today?I read around was used for call/callvirt IL.

Reply:
Right, it is fine to delete these checks.

Reply:
fixed